rasa/core/featurizers/tracker_featurizers.py

```diff
@@ -178,14 +183,60 @@ def training_states_and_actions(
         Returns:
             A tuple of list of states and list of actions.
         """
+
+        rasa.shared.utils.io.raise_deprecation_warning(
+            "'training_states_and_actions' is being deprecated in favor of "
+            "'training_states_and_labels'."
+        )
+
+        return self.training_states_and_labels(
+            trackers, domain, omit_unset_slots=omit_unset_slots
+        )
+
+    def training_states_and_labels(
```

Comment:
there's still 12 calls to `training_states_and_actions` in the code base although it is deprecated now. I assume these should be changed to calling `training_states_and_labels` now otherwise we'll get a lot of warnings.

Reply:
In general, it might be worth asking: why keep `training_states_and_actions` at all?